lighthaus.py

```diff
@@ -23,69 +22,93 @@ def __init__(
             initial_scroll_speed: float = 0,
             sleep_time: float = 0.01,
             brightness: float = 1.0,
+            fade_in_time: float = 5.0,
```

Comment:
I think the defaults for fade in / out should be longer. I feel strongly that the fade out should be longer than 20 seconds.